sycl/include/sycl/ext/intel/esimd/detail/math_intrin.hpp

```diff
@@ -72,23 +72,6 @@ template <typename T, int N>
 __ESIMD_INTRIN __ESIMD_raw_vec_t(T, N)
     __spirv_ocl_native_powr(__ESIMD_raw_vec_t(T, N), __ESIMD_raw_vec_t(T, N));
 
-template <typename T, int N>
-__ESIMD_INTRIN __ESIMD_raw_vec_t(T, N)
```

Comment:
instead what if we SFINAE these to only work when N==1, so we use the compiler generated ones by default?

Reply:
Whatever, I'll deal with this later. Will approve/merge for now.